docs/release-notes/1.17.1.md

```diff
@@ -0,0 +1,3 @@
+## Fixed Bugs
+
+- Update **cniBinDir** for k3d clusters. This change ensures compatibility with versions of k3s that are greater than 1.32.
```

Comment:
`Update **cniBinDir** for k3d clusters.` sounds like an instruction. Is this intended?

Reply:
I'll change it to "We've updated"